src/display/text_layer.js

```diff
@@ -182,9 +495,12 @@ var renderTextLayer = (function renderTextLayerClosure() {
    * @param {HTMLElement} container
    * @param {PageViewport} viewport
    * @param {Array} textDivs
+   * @param {boolean} enhanceTextSelection
+   * @param {boolean} enhanceOnClick
```

Comment:
As far as I can tell, with the exception of the tests, there's no reason why you'd ever not want the "expand-on-click" behaviour in the viewer.
Hence I'm wondering if it wouldn't make more sense to invert this parameter to instead be `false` by default, and requiring the user (i.e. the test driver) to pass in something to *explicitly* disable the "expand-on-click" behaviour, e.g. with a parameter like `disableExpandOnClick`, `forceExpand`, or similar.